<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="nx">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ny">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prec">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1e-8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="steplim">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QComboBox" name="method">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Метод Зейделя</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Метод min невязок</string>
     </property>
    </item>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>681</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="v_tab">
     <attribute name="title">
      <string>V</string>
     </attribute>
     <widget class="QTableWidget" name="v_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>581</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>j</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>i</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>Y\X</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="uv2_tab">
     <attribute name="title">
      <string>V_2 (U)</string>
     </attribute>
     <widget class="QTableWidget" name="uv2_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>581</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>i</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>j</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>Y\X</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="vv2_tab">
     <attribute name="title">
      <string>V-V_2(U)</string>
     </attribute>
     <widget class="QTableWidget" name="vv2_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>581</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>i</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>j</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>Y\X</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="r_tab">
     <attribute name="title">
      <string>r</string>
     </attribute>
     <widget class="QTableWidget" name="r_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>581</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>i</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>j</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>Y\X</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Итераций:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Точность метода:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимум разности V и V_2 (U)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>в</string>
    </property>
   </widget>
   <widget class="QComboBox" name="task">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Тестовая</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Основная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Тестовая (без угла)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>n</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ε</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLabel" name="s">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="eps_s">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="maxd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="maxd_point">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Норма невязки:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxr">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
